views/show.erubis

```diff
@@ -10,7 +10,7 @@
 
       <ul class="list list-bullet">
         <% @case.files.each do |file| %>
-          <li><a href="<%= file.s3_url %>" title="<%= file.name %>"><%= file.name %></a></li>
+          <li><a href="<%== file.s3_url %>" title="<%== file.name %>"><%== file.name %></a></li>
```

Comment:
That's `<%===` in Erubis.  `<%==` is escaped.  See section 2-3 'Escape':

http://www.kuwata-lab.com/erubis/users-guide.02.html